jenkins/run

```diff
@@ -53,16 +64,39 @@ fi
 
 _image=${IMAGE:-${_image_repo}/pbench-${_image_role}-${_image_kind}:${_branch_name}}
 
-podman run \
+if [ -n "${CONTAINER_HOST}" ]; then
+    # Since CONTAINER_HOST is defined, when we want to run another container, do
```

Comment:
Can you add to this something like, "this is the core mechanism that allows CI jobs to run sub-sets of the system in separate containers"?